runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkStreamingTranslationContext.java

```diff
@@ -86,57 +86,52 @@ public void setOutputDataStream(PValue value, DataStream<?> set) {
    *
    * @param currentTransform
    */
-  public void setCurrentTransform(AppliedPTransform<?, ?, ?> currentTransform) {
+  void setCurrentTransform(AppliedPTransform<?, ?, ?> currentTransform) {
     this.currentTransform = currentTransform;
   }
 
-  public <T> Coder<WindowedValue<T>> getWindowedInputCoder(PCollection<T> collection) {
```

Comment:
This one is the main reason for this PR because it's really used as `getWindowedCoder` for inputs and outputs.